src/common/bsod_gui.cpp

```diff
@@ -153,12 +153,8 @@ void fatal_error(const char *error, const char *module) {
     /// Decision tree to define error code
     using namespace Language_en;
     /// TODO share these strings (saves ~100 B of binary size)
-    if (strcmp(MSG_INVALID_EXTRUDER_NUM, error) == 0) {
-        *perror_code_short = 0;
-    } else if (strcmp("Emergency stop (M112)", error) == 0) {
+    if (strcmp("Emergency stop (M112)", error) == 0) {
         *perror_code_short = 510;
-    } else if (strcmp("Inactive time kill", error) == 0) {
-        *perror_code_short = 0;
```

Comment:
Why MSG_INVALID_EXTRUDER_NUM and Inactive time kill cases are removed?

Reply:
I know that it does nothing at the moment, but I would consider leaving it there in case perror_code_short becomes defined for those error states.

Reply:
It seems to me, that leaving it there with unassigned error code is not very clean. Once new error codes are assigned, then new cases will be created...